### DISCUSSION TITLE
Unable to install Pyaudio / Pyaudio Error

### DISCUSSION THREAD

**Discussion by Dipeshpal** [Dipeshpal/Jarvis_AI#13] [Q&A]
If you are unable to install this then follow-

1. Download ".whl" file from https://pypi.org/project/PyAudio/#files
2. Then after downloading, locate the file and open cmd in the same directory the type: "pip install PyAudio-0.2.11-cp27-cp27m-win32.whl" (You might need to change file name)

**Comment by Akul2010** [comment]
you could also install pipwin and then install pyaudio

```
pip install pipwin
pipwin install pyaudio
```
